exoscale/datasource_exoscale_compute_instance_list.go

```diff
@@ -14,6 +15,64 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+const (
+	filterStringPropName = "match"
+	filterRegexPropName  = "regex_match"
```

Comment:
```suggestion
	filterStringPropName = "name"
```
One one _could_ do, to avoid the duplication of `name` and `name_regex` is detect whether the `name` use the the well-known (regex) `/.../<flags>` syntax: if it does: regexp match, otherwise, regular match

WDYT ?

Reply:
changed it, I'm much happier with it now